spec/requests/reviews_spec.rb

```diff
@@ -75,4 +73,36 @@
     end
   end
 
+  describe "EDIT /update" do
+    it "will update a review" do
+
+      user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+      school1 = School.create(name: "LEARN academy")
+
+        review_params = {
+          review: {
+          title: "My Second Review",
+          date_posted: "August 19, 2022",
+          review_text: "Test review text",
+          user_id: user.id,
+          school_id: school1.id
+        }
+      }
+      post '/reviews', params: review_params
+
+      review = Review.first
+      update_params = { review: { title: "My Second Review" } }
```

Comment:
This doesn't look any different than your initial title on line 84.  You should be checking for a change here.